### PR TITLE
Resources page update on video order + a new LU video

### PR DESCRIPTION
**Pull request by szabozoltan69**
Refers to https://github.com/IFRCGo/go-web-app/issues/1370

### CODE REVIEW

File: .changeset/lucky-coats-help.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"go-web-app": patch
+---
+
+Resources page update on video order + a new LocalUnit video
```
Comment (frozenhelium) [inline .changeset/lucky-coats-help.md]: Let's make the change-log easier for the users to understand

```suggestion
Update resources page
  - Add a new video for LocalUnits
  - Update ordering of videos
```

**Review by frozenhelium — CHANGES_REQUESTED** [review]

**Review by tnagorra — APPROVED** [review]

File: app/src/views/Resources/VideoList/i18n.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
         "videoCarouselHeadingMontandon": "Montandon – the Global Crisis Data Bank",
         "videoCarouselSubHeadingMontandon": "Mapping global crises' historical data. The IFRC GO Platform uses Montandon, the Global Crisis Data Bank, the world's largest repository of information on past natural hazards and their impact.",
         "videoCarouselHeading3w": "3W (who-what-where) data collection for emergencies",
-        "videoCarouselHeading3wsub": "View this video tutorial about the 3W for emergencies tool. The video demonstrates the data collection and display activity of who-what-where data."
+        "videoCarouselSubHeading3w": "View this video tutorial about the 3W for emergencies tool. The video demonstrates the data collection and display activity of who-what-where data.",
```
Comment (tnagorra) [inline app/src/views/Resources/VideoList/i18n.json]: We have duplicate keys for "videoCarouselHeading3W" and "videoCarouselSubHeading3W"
Also, "videoCarouselSubHeading3WInstructional" is not used.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/src/views/Resources/VideoList/i18n.json (hunk 2 above)
Comment (frozenhelium) [reply app/src/views/Resources/VideoList/i18n.json]: @szabozoltan69 could you take a look at this and verify the correct behavior?

Comment (szabozoltan69) [reply app/src/views/Resources/VideoList/i18n.json]: 3W != 3w, but I fixed it with a more verbose naming. And removed unused variable.

**Review by samshara — APPROVED** [review]

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**